### PR TITLE
Setting meeting validation

### PR DESCRIPTION
**Pull request by DanitzaMachicado**
- Display correctly date on picker date
- Fix warning on picker date

### CODE REVIEW

File: src/main/webapp/assets/js/settingMeeting.js [hunk 1]
```diff
@@ -37,8 +37,8 @@ var SettingMeeting = function() {
 	var setValues = function() {
 		var time = moment(dateField.val()).format("HH:mm");
 		timeField.val(time);
-		var value = moment(dateField.val()).format("D MMMM, YYYY");
-		dateField.val(value);
+
```
Comment (alexandercastrojala) [inline src/main/webapp/assets/js/settingMeeting.js]: delete white line

**Review by alexandercastrojala — COMMENTED** [review]
only still one comment..after looks well ...

**Review by alexandercastrojala — APPROVED** [review]
look well..

File: src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java [hunk 2]
```diff
@@ -64,4 +66,17 @@ private int calculateWidth(int days) {
 
 		return width;
 	}
+	
+	/**
+	 *
+	 * @return
+	 */
+	public java.time.LocalDate getDate() {
```
Comment (MirkoTV) [inline src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java:70]: Please complete these comments or delete them

**Review by MirkoTV — CHANGES_REQUESTED** [review]
Good Job!

**Review by MirkoTV — APPROVED** [review]

**Pull request merged**